cookiecutter/repository.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+"""Cookiecutter repository functions."""
+from __future__ import unicode_literals
+import os
+import re
+
+from .exceptions import RepositoryNotFound
+from .vcs import clone
+
+BUILTIN_ABBREVIATIONS = {
```

Comment:
As mentioned above it might make sense to have this as a default in **config.py**.